app/serializers/webhook/article_serializer.rb

```diff
@@ -0,0 +1,11 @@
+module Webhook
+  class ArticleSerializer
+    include FastJsonapi::ObjectSerializer
+    set_type :article
+    belongs_to :user
+    attributes :archived, :automatically_renew, :body_html, :body_markdown, :cached_tag_list,
```

Comment:
I'm looking at the list of attributes and wondering if there shouldn't be the ID as well. Generally speaking I'm not in favor of sending out internal IDs, but we expose them in the Articles API already, and without an ID it'll be difficult to associate an event to the resource in the API.

The list of fields in general should probably be the same as https://github.com/thepracticaldev/dev.to/blob/master/app/views/api/v0/articles/show.json.jbuilder

but I defer to @benhalpern to have a look at what we should or shouldn't send in the article payload outside of the system.

For example `removed_for_abuse` sounds like an internal attribute, but I might be wrong

Reply:
I'm also not a fan of exposing ids, and the attributes list is pretty random currently. It's a good idea to make it the same as in the `articles/show.json.builder`.
I'm also interested in @ben's and the rest of the team's opinion on using `fast_jsonapi` for serialization.

Reply:
I think using `fast_jsonapi` is neat and fine but is there a security concern with `jbuilder`?

Reply:
+1 for [`fast_jsonapi`](https://github.com/Netflix/fast_jsonapi)

Reply:
@maestromac I think the reason @lightalloy chose `fast_jsonapi` it's because serialization is much faster: https://buttercms.com/blog/json-serialization-in-rails-a-complete-guide

Reply:
Yep, that was the reason.

Reply:
Yup, works for me. This looks like a clean approach alongside the performance.